### PR TITLE
Chec 3259 shipping zone api (#1)

### PR DESCRIPTION
**Pull request by sandersjessica**
* Create the Shipping Zone Api including service and models for shipping zone, province and country

* Convert countries and provinces to collections not arrays

### CODE REVIEW

**Comment by rickywiens** [comment]
Looks good to me

File: src/Models/Country.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    /**
+     * @var string
+     */
+    protected $code;
+
+    /**
+     * @var int
+     */
+    protected $id;
```
Comment (nathankoop) [inline src/Models/Country.php]: i'm partial to id being the first property

File: src/Models/Province.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    /**
+     * @var int
+     */
+    protected $countryId;
+
+    /**
+     * @var int
+     */
+    protected $id;
```
Comment (nathankoop) [inline src/Models/Province.php]: same as above

File: src/Models/ShippingZone.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace BoldApps\ShopifyToolkit\Models;
+
+use BoldApps\ShopifyToolkit\Contracts\Serializeable;
+
+class ShippingZone implements Serializeable
+{
+
+    /**
```
Comment (nathankoop) [inline src/Models/ShippingZone.php]: missing the ID field for this entity

**Comment by jerairrest** [comment]
@nathankoop is this good or not? You approved then left comments

File: src/Models/ShippingZone.php (hunk 3 above)
Comment (digiexchris) [inline src/Models/ShippingZone.php]: depends on if shopify returned an id or not

Comment (rickywiens) [inline src/Models/ShippingZone.php]: According to the Shopify API Documentation this entity has an id.

**Comment by nathankoop** [comment]
I'd like to see the id added, but I'm fine if you merge it now and fix later

**Comment by sandersjessica** [comment]
I added the id to the Shipping zone, and made the id's the first property in the model! Should be good to go now.

**Pull request merged**